src/patternfly/components/DualListSelector/dual-list-selector.scss

```diff
@@ -0,0 +1,173 @@
+  --pf-c-dual-list-selector--m-chosen__menu--GridArea: pane-menu-c;
+  --pf-c-dual-list-selector--GridTemplateColumns--pane--MinMax--min: #{pf-size-prem(200px)};
+  --pf-c-dual-list-selector--GridTemplateColumns--pane--MinMax--max: #{pf-size-prem(450px)};
+
+  // Header
+  --pf-c-dual-list-selector__header--MarginBottom: var(--pf-global--spacer--sm);
+
+  // Title
+  --pf-c-dual-list-selector__title-text--FontWeight: var(--pf-global--FontWeight--bold);
+
+  // Tools
+  --pf-c-dual-list-selector__tools--MarginBottom: var(--pf-global--spacer--md);
+  --pf-c-dual-list-selector__tools-filter--tools-actions--MarginLeft: var(--pf-global--spacer--sm);
+
+  // Menu
+  --pf-c-dual-list-selector__menu--BoxShadow: var(--pf-global--BoxShadow--md);
+  --pf-c-dual-list-selector__menu--MinHeight: #{pf-size-prem(200px)};
+  --pf-c-dual-list-selector__menu--MaxHeight: #{pf-size-prem(320px)};
+
+  // List item
+  --pf-c-dual-list-selector__list-item--BackgroundColor: transparent;
+  --pf-c-dual-list-selector__list-item--hover--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+  --pf-c-dual-list-selector__list-item--focus-within--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+  --pf-c-dual-list-selector__list-item--m-selected--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+
+  // Item
+  --pf-c-dual-list-selector__item--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-dual-list-selector__item--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-dual-list-selector__item--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-dual-list-selector__item--PaddingLeft: var(--pf-global--spacer--md);
+
+  // Item text
+  --pf-c-dual-list-selector__item-text--Color: var(--pf-global--Color--100);
+  --pf-c-dual-list-selector__list-item--m-selected__item-text--Color: var(--pf-global--active-color--100);
+
+  // Status
+  --pf-c-dual-list-selector__status--MarginBottom: var(--pf-global--spacer--sm);
+  --pf-c-dual-list-selector__status-text--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-dual-list-selector__status-text--Color: var(--pf-global--Color--200);
+
+  // Controls
+  --pf-c-dual-list-selector__controls--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-dual-list-selector__controls--PaddingLeft: var(--pf-global--spacer--md);
+
+  display: grid;
+  grid-template-areas:
+    "pane-header . pane-header-c"
+    "pane-tools . pane-tools-c"
+    "pane-status . pane-status-c"
+    "pane-menu controls pane-menu-c";
+  grid-template-columns:
+    minmax(var(--pf-c-dual-list-selector--GridTemplateColumns--pane--MinMax--min), var(--pf-c-dual-list-selector--GridTemplateColumns--pane--MinMax--max))
+    min-content
+    minmax(var(--pf-c-dual-list-selector--GridTemplateColumns--pane--MinMax--min), var(--pf-c-dual-list-selector--GridTemplateColumns--pane--MinMax--max));
+  grid-template-rows: repeat(3, auto) auto;
+}
+
+.pf-c-dual-list-selector__pane {
+  display: contents;
```

Comment:
Haha I'm pretty proud of that 🥳 It was the best way I found to align the controls with the list boxes.

Though since it's a little weird and we can't apply styles directly to `__pane` now, we could also make the list heights a static thing assignable via a variable, and make the controls element that height by sharing the variable, and bottom align the controls in the overall layout, then center align the buttons in the control element. You lose the fluid height adjustment of the lists, but I think that would be OK. WDYT?